src/payment/strategies/paypal-commerce/paypal-commerce-payment-processor.ts

```diff
@@ -143,6 +133,19 @@ export default class PaypalCommercePaymentProcessor {
         return this._hostedFields.submit(options);
     }
 
+    getValidationStateHostedFields(): { isValid: boolean; fields: PaypalCommerceHostedFieldsState['fields'] } {
```

Comment:
`getHostedFieldsValidationState` might read better.

Reply:
Yeah, thank you!